swagger-ui/dist/zipskee-v1.swagger.yaml

```diff
@@ -136,6 +136,8 @@ paths:
                type: string
              password:
                type: string
+             passwordConfirmation:
```

Comment:
It has to be passed to the server. Devise requires the following both password and password confirmation to create a new resource 'user'. @michaelsobota